rom-hacking/scripting/animation-commands/ac-index-anime.md

```diff
@@ -0,0 +1,94 @@
+# (870) _AC_INDEX_ANIME
+
+## Effect
+
+Animates the actor based on the given animation. The list of animations can be found at the bottom of this page.
+
+If the second argument is provided, the animation takes two seconds.
+
+## Syntax
+
+```c
+_AC_INDEX_ANIME(animation, two_seconds?)
+```
+
+| Argument | Description | Types | Required |
+| - | - | - | - |
+| **animation** | The animation to perform | Float | Required |
+| **two_seconds** | Whether the animation should take 2 seconds | Any | Optional |
+
+## Example
+
+```c
+anm_dummy:
+_AC_INDEX_ANIME(41)
+_AC_INDEX_ANIME_WAIT()
+_ACMD_END()
+```
+
+The above script will make the actor spin around in the air.
+
+## List of animations
```

Comment:
Great addition with the GIFs! The names of a few of these aren't 100% accurate, here's a list of the animation clips: https://docs.google.com/spreadsheets/d/19BiWB92aD1cpIbe-YT8B0YjMbNqSoriwfAHefZV00FY/edit#gid=0. You can name them however you'd like, the names there are just the internal names and might help figuring out a few you seemed to be confused on.

It might be worth adding an extra column for if an animation is exclusive to the player (41-45). Or exclusive to a specific chibi (Bow for nurses with 19).